packages/core/src/provider/initPlugins.ts

```diff
@@ -63,12 +63,26 @@ export function getPluginAPI({
     );
   };
 
+  const exposed: Array<{ id: string | symbol; api: any }> = [];
+
+  const expose = (id: string | symbol, api: any) => {
+    exposed.push({ id, api });
```

Comment:
Should we freeze the `api` to avoid useExposed modifying it?

Reply:
Yes, we should do this, and the RsbuildPluginAPI object should also be readonly.

But I want to avoid introducing runtime overhead, I will add TypeScript `ReadOnly<T>` first.

Reply:
I reconsidered and it seems can be allowed to modify the exposed object.

For example, pluginA can expose an object that contains some properties, and maybe pluginA want pluginB to modify the properties for communication in some cases.